cluster-operator/src/main/resources/cluster-roles/020-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -8,6 +8,7 @@ rules:
 - apiGroups:
   - ""
   resources:
+  # The cluster operator needs to access and manage service accounts to grant applications cluster permissions
```

Comment:
isn't "applictions" too much generic? could we say "Strimzi components" or "other components", we are not talking about generic (user) applications.

Reply:
Very true, will amend.